Users/coafile.rst

```diff
@@ -93,7 +93,7 @@ This is the same file without section inheritance::
 All settings must be part of a section, so don't do this for implicit
 inheritance (this is also deprecated behavior). Implicit inheritance
 was leading to a section automatically getting inherited to all other
-sections without semantically making sense.
+sections without semantically making sense.::
```

Comment:
@0x48piraj Remove trailing period, then it will look fine 👍

Reply:
But, everywhere the same pattern is followed.

Reply:
Yeah, that's what I'm saying, everywhere the pattern should be same.But in your case, it should be like this
`sections without semantically making sense::`

Reply:
There's a full-stop at the last of sentence, so I left it. Should I remove the full-stop ? :blush:

Reply:
yes, that's what I'm talking about.

Reply:
See for now, it looks like [this](https://screenshots.firefox.com/UvhCTZ76mDP3fb8f/github.com) 😃

Reply:
yes, and it should look like this `making sense:`

Reply:
No @khanchi97 , `making sense:` reverts the whole issue back to zero :wink:

Reply:
And, `making sense::` does - `making sense:`
I think my first commit was OK. :blush:

Reply:
If you code like this `making sense::` then it will look like `making sense:` that you have done already.Now you only have to squash your commits 👍

Reply:
**I am sorry, But I'm on Web Interface, I will take care from the next time ! :confused:**
A maintainer can merge and squash the commits easily.
![](https://user-images.githubusercontent.com/5800726/32701602-c684d926-c7fe-11e7-9077-0c21033d9574.png)
> Reference : https://stackoverflow.com/questions/35948131/github-squash-commits-from-web-interface-on-pull-request-after-review-comments